tools/gsc-filter_cells/filter_cells.xml

```diff
@@ -0,0 +1,189 @@
+            <output name="output_metada" file="no-filtering.meta" ftype="tabular"/>
+        </test>
+        <test>
+            <param name="input" value="input.csv" ftype="txt"/>
+            <param name="sep" value='comma' />
+            <param name="absolute_genes" value="5"/>
+            <param name="absolute_counts" value="1000"/>
+            <output name="pdfplot" file="absolute_gene-and-counts.pdf" ftype="pdf"/>
+            <output name="output" file="absolute_gene-and-counts.tab" ftype="tabular"/>
+            <output name="output_metada" file="absolute_gene-and-counts.meta" ftype="tabular"/>
+        </test>
+        <test>
+            <param name="input" value="input.csv" ftype="txt"/>
+            <param name="sep" value='comma' />
+            <param name="absolute_genes" value="5"/>
+            <output name="pdfplot" file="absolute_gene-only.pdf" ftype="pdf"/>
+            <output name="output" file="absolute_gene-only.tab" ftype="tabular"/>
+            <output name="output_metada" file="absolute_gene-only.meta" ftype="tabular"/>
+        </test>
+        <test>
+            <param name="input" value="input.csv" ftype="txt"/>
+            <param name="sep" value='comma' />
+            <param name="absolute_counts" value="1000"/>
+            <output name="pdfplot" file="absolute_counts-only.pdf" ftype="pdf"/>
+            <output name="output" file="absolute_counts-only.tab" ftype="tabular"/>
+            <output name="output_metada" file="absolute_counts-only.meta" ftype="tabular"/>
+        </test>
+    </tests>
+    <help>
+
+**What it does**
+
+The tools takes a table of gene (rows) expression values (as number of reads aligned to genes)
+in single cell RNAseq sequencing libraries (columns) and filters out cells with low number
+of detected genes and/or cells with low number of aligned reads.
+
+Cutoffs can be applied to absolute numbers of aligned reads or of detected genes, or to
+percentile thresholds for these variables.
+
+For both absolute or percentile thresholds, only cells exclusively below
+these threshold are excluded (cell cutoffs do not include the threshold values).
+
+If you choose to combine cutoffs for both the number of detected genes
+and the total number of aligned reads, then you have 2 options: either exclude libraries that
+do not satisfy one OR the other threshold (Union) or exclude libraries that do not satisfy
+both thresholds (Intersection).
+
+Specifying a value both for an absolute and an percentile threshold of a variable
+(Number of detected genes or Number of aligned counts) is not consistent. In this
+situation, the tools *does not* filter cells with respect to the corresponding variable threshold.
+If a 0 is applied both for an absolute and an percentile threshold of a variable, then
+this variable is not used to filter the cells.
+
+The tools returns a gene expression table for cells that were retained, a metadata table
```

Comment:
Ici aussi je mettrais "the gene table counts" à la place de "a gene expression table" pour être plus spécifique

Reply:
pareil, gene table counts est moins parlant pour un biologiste que gene expression table. `gene counts` peut être interprété stricto sensu c'est à dire "comptage de gènes".